aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -8,23 +8,14 @@ export const ThemeModeToggle = () => {
   const label = darkMode.value ? 'Light Mode' : 'Dark Mode';
   const icon = darkMode.value ? <IconSun /> : <IconMoon />;
   return (
-    <Box align="start">
-      <Button onClick={() => darkMode.toggle()}>
-        <Box
-          background="blue!"
-          direction="row"
-          gap="xsmall"
-          justify="center"
-          overflow="hidden"
-          pad={{ vertical: 'small', horizontal: 'medium' }}
-          responsive={false}
-          round="xlarge"
-          width="small"
-        >
-          {icon}
-          {label}
-        </Box>
-      </Button>
+    <Box width={{ max: 'small' }}>
+      <Button
+        color={{ dark: 'blue!', light: 'white' }}
```

Comment:
```suggestion
        color="background-front"
```

Reply:
I'm going to leave as is since it looks like Chris already updated how this Button appears in the Footer -- we can update in a separate PR